tests/integration/aws_s3/test_aws_s3.py

```diff
@@ -13,18 +13,20 @@
 # limitations under the License.
 #
 import os
-import random
-import tempfile
 import uuid
+from os.path import abspath, dirname, join
```

Comment:
These tests already use `pathlib`. Is there any justification to also using `os.path`? Also, in such a case it's better to `import os.path` and use the fully qualified name so that it's clear what these functions are.

Reply:
Naturally, I was referring to all occurrences of this, not just in this one test file.

Reply:
I can not find another pathlib import